<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <!-- interface-requires gtk+ 3.0 -->
  <object class="GtkImage" id="add_image">
    <property name="visible">True</property>
    <property name="can_focus">False</property>
    <property name="stock">gtk-add</property>
  </object>
  <object class="GtkDialog" id="unused">
    <property name="can_focus">False</property>
    <property name="border_width">5</property>
    <property name="type_hint">dialog</property>
    <child internal-child="vbox">
      <object class="GtkBox" id="toplevel">
        <property name="can_focus">False</property>
        <property name="orientation">vertical</property>
        <property name="spacing">2</property>
        <child internal-child="action_area">
          <object class="GtkButtonBox" id="dialog-action_area1">
            <property name="can_focus">False</property>
            <property name="layout_style">end</property>
            <child>
              <object class="GtkButton" id="cancel_button">
                <property name="label">gtk-cancel</property>
                <property name="use_action_appearance">False</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="use_action_appearance">False</property>
                <property name="use_stock">True</property>
                <signal name="activate" handler="gw_addvocabularywindow_cancel_cb" object="toplevel" swapped="no"/>
                <signal name="clicked" handler="gw_addvocabularywindow_cancel_cb" object="toplevel" swapped="no"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="add_button">
                <property name="label" translatable="yes">_Add Vocabulary Word</property>
                <property name="use_action_appearance">False</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="use_action_appearance">False</property>
                <property name="image">add_image</property>
                <property name="use_underline">True</property>
                <signal name="activate" handler="gw_addvocabularywindow_add_cb" object="toplevel" swapped="no"/>
                <signal name="clicked" handler="gw_addvocabularywindow_add_cb" object="toplevel" swapped="no"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="pack_type">end</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="content_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="border_width">6</property>
            <property name="orientation">vertical</property>
            <property name="spacing">5</property>
            <property name="homogeneous">True</property>
            <child>
              <object class="GtkBox" id="box1">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="spacing">8</property>
                <child>
                  <object class="GtkLabel" id="kanji_label">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="xalign">0</property>
                    <property name="label" translatable="yes">Kanji:</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">True</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEntry" id="kanji_entry">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="invisible_char">●</property>
                    <property name="caps_lock_warning">False</property>
                    <property name="placeholder_text">Enter your Kanji here...</property>
                  </object>
                  <packing>
                    <property name="expand">True</property>
                    <property name="fill">True</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkBox" id="furigana_box">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="spacing">8</property>
                <child>
                  <object class="GtkLabel" id="furigana_label">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="xalign">0</property>
                    <property name="label" translatable="yes">Furigana:</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">True</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEntry" id="furigana_entry">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="invisible_char">●</property>
                    <property name="placeholder_text">Enter your Furigana here...</property>
                  </object>
                  <packing>
                    <property name="expand">True</property>
                    <property name="fill">True</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkBox" id="definitions_box">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="spacing">8</property>
                <child>
                  <object class="GtkLabel" id="definitions_label">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="xalign">0</property>
                    <property name="label" translatable="yes">Definitions:</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">True</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkEntry" id="definitions_entry">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="invisible_char">●</property>
                    <property name="placeholder_text">Enter your Definitions here...</property>
                  </object>
                  <packing>
                    <property name="expand">True</property>
                    <property name="fill">True</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkBox" id="list_box">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="spacing">8</property>
                <child>
                  <object class="GtkLabel" id="list_label">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="xalign">0</property>
                    <property name="label" translatable="yes">List:</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">True</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBox" id="vocabulary_list_combobox">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="has_entry">True</property>
                    <child internal-child="entry">
                      <object class="GtkEntry" id="combobox-entry">
                        <property name="can_focus">False</property>
                        <property name="placeholder_text">Enter your Vocabulary List Name here...</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">True</property>
                    <property name="fill">True</property>
                    <property name="position">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">3</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="padding">12</property>
            <property name="position">1</property>
          </packing>
        </child>
      </object>
    </child>
    <action-widgets>
      <action-widget response="0">cancel_button</action-widget>
      <action-widget response="0">add_button</action-widget>
    </action-widgets>
  </object>
  <object class="GtkSizeGroup" id="entry_sizegroup">
    <widgets>
      <widget name="kanji_entry"/>
      <widget name="furigana_entry"/>
      <widget name="definitions_entry"/>
      <widget name="vocabulary_list_combobox"/>
    </widgets>
  </object>
  <object class="GtkSizeGroup" id="label_sizegroup">
    <property name="mode">both</property>
    <widgets>
      <widget name="kanji_label"/>
      <widget name="furigana_label"/>
      <widget name="definitions_label"/>
      <widget name="list_label"/>
    </widgets>
  </object>
</interface>
